src/2024h2/ergonomic-rc.md

```diff
@@ -102,7 +102,7 @@ The work here is proposed by Jonathan Kelley on behalf of Dioxus Labs. We have f
 | Author RFC                 | TBD                 | ![Funded][] |
 | Design meeting             | ![Team][] [Lang] |
 | Accept RFC                 | ![Team][] [Lang] |
-| Nightly implementation     | TBD                 | ![Funded][] |
+| Nightly implementation     | [spastorino]        | ![Funded][] |
```

Comment:
My understanding of `[Funded]` here is that I'm funded by someone to do the work.

Reply:
I would leave it, it is meant to mean "funded by someone" :)

Reply:
I've been a bit unsure about whether to highlight being 'funded' or not, I want to kind of normalize talking about this kind of thing